Library/ViewModels/DashboardViewModel.swift

```diff
@@ -87,25 +102,62 @@ public protocol DashboardViewModelType {
 public final class DashboardViewModel: DashboardViewModelInputs, DashboardViewModelOutputs,
   DashboardViewModelType {
 
-    public init() {
+  private let selectedProjectProperty: SignalProducer<Param?, NoError>
```

Comment:
It might be good to add comments to these detailing what they do, particularly because this is a slight deviation from our other viewModels.